src/routes/LocationPage/index.js

```diff
@@ -93,13 +89,18 @@ class LocationPage extends React.Component {
   }
 }
 
+const mapLanguageToUrl = (location, language, id) => (
+  id !== '' ? `/${location}/${language}/redirect?id=${id}` : `/${location}/${language}`
```

Comment:
Use a constant here instead of an empty string. What does empty string mean? Maybe undefined is better here.